src/controllers/authentication/user.js

```diff
@@ -9,52 +12,57 @@ import { responseFormat, errorResponseFormat } from '../../utils/index';
  * @returns {object|void} response object
  */
 export const login = (req, res) => {
-  const {
-    id, fullName, bio, email, username, role
-  } = req.user;
-  return res.status(200).json(responseFormat({
-    status: 'success',
-    data: {
-      token: Auth.generateToken({
-        id, fullName, bio, email, username, role
-      })
-    },
-  }));
+  const { id, fullName, bio, email, username, role } = req.user;
+  return res.status(200).json(responseFormat({ status: 'success', data: { token: Auth.generateToken({ id, fullName, bio, email, username, role }) }, }));
 };
 
 export const createUser = async (req, res) => {
   try {
     const { body } = req;
-    const user = await User.create({ ...body });
+    const verificationToken = uuid();
+    const user = await User.create({ ...body, verificationToken });
     const { id, username, email, role, fullName, bio } = user;
 
+    let url = `http://${process.env.HOST}:${process.env.PORT}/api/v1/verify`;
+
+    if (process.env.NODE_ENV === 'production') { url = `https://${process.env.HOST}:${process.env.PORT}/api/v1/verify`; }
```

Comment:
Let us learn to separate code and put them where they should belong. This should not be here.

Reply:
It has been resolved. Thanks